### PR TITLE
[INLONG-5235][Dashboard] Support users to modify passwords and modify personal keys

### PR DESCRIPTION
**Pull request by bluewang**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes https://github.com/apache/inlong/issues/5235

### CODE REVIEW

File: inlong-dashboard/src/components/Layout/NavWidget/KeyModel.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          accountType: result.accountType,
+        });
+      },
+    },
+  );
+
+  const onOk = async () => {
+    const values = await form.validateFields();
+    const data = { ...userData, ...values };
+    console.log(data);
```
Comment (leezng) [inline inlong-dashboard/src/components/Layout/NavWidget/KeyModel.tsx:99]: remove console

File: inlong-dashboard/src/components/Layout/NavWidget/index.tsx [hunk 2]
```diff
@@ -17,21 +17,31 @@
...
-import React from 'react';
+import React, { useState } from 'react';
 import { Dropdown, Menu } from 'antd';
 import { useHistory, useSelector, useDispatch, useRequest } from '@/hooks';
 import { State } from '@/models';
 import { useTranslation } from 'react-i18next';
 // import { FileTextOutlined } from '@/components/Icons';
 import LocaleSelect from './LocaleSelect';
 import styles from './index.module.less';
+import PassModel from './PasswordModel';
```
Comment (leezng) [inline inlong-dashboard/src/components/Layout/NavWidget/index.tsx:28]: rename to `PasswordModel`

File: inlong-dashboard/src/components/Layout/NavWidget/PasswordModel.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+          accountType: result.accountType,
+        });
+      },
+    },
+  );
+
+  const onOk = async () => {
+    const values = await form.validateFields();
+    if (values.newPassword !== values.confirmPassword) {
+      message.warning(i18n.t('components.Layout.NavWidget.Remind'));
```
Comment (leezng) [inline inlong-dashboard/src/components/Layout/NavWidget/PasswordModel.tsx:98]: The logic of checking whether the password is consistent before and after can refer to this： https://github.com/apache/inlong/blob/master/inlong-dashboard/src/components/AccessHelper/FieldsConfig/businessFields.tsx#L196

**Review by leezng — CHANGES_REQUESTED** [review]

File: inlong-dashboard/src/components/Layout/NavWidget/index.tsx [hunk 4]
```diff
@@ -17,21 +17,31 @@
...
+import React, { useState } from 'react';
 import { Dropdown, Menu } from 'antd';
 import { useHistory, useSelector, useDispatch, useRequest } from '@/hooks';
 import { State } from '@/models';
 import { useTranslation } from 'react-i18next';
 // import { FileTextOutlined } from '@/components/Icons';
 import LocaleSelect from './LocaleSelect';
 import styles from './index.module.less';
+import PasswordModel from './PasswordModel';
+import KeyModel from './KeyModel';
```
Comment (healchow) [inline inlong-dashboard/src/components/Layout/NavWidget/index.tsx:29]: Does those `Model` need to be changed to `Modal`?

**Review by healchow — APPROVED** [review]

**Review by leezng — APPROVED** [review]

**Pull request merged**